Lib/unittest/mock.py

```diff
@@ -2422,15 +2422,12 @@ def __set__(self, obj, val):
 
 
 def seal(mock):
-    """Disable the automatic generation of "submocks"
+    """Disable the automatic generation of child mocks.
 
     Given an input Mock, seals it to ensure no further mocks will be generated
     when accessing an attribute that was not already defined.
 
-    Submocks are defined as all mocks which were created DIRECTLY from the
-    parent. If a mock is assigned to an attribute of an existing mock,
-    it is not considered a submock.
-
+    The operation seals the mock passed in and all its children recursively.
```

Comment:
Can you add here an explanation of what "child mocks" are? The documentation does not clarify exactly what a child mock is and this can be confusing.